hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -29,9 +29,68 @@ <h2 class="formH2">Complete your application</h2>
                         Get started
                     </a>
                 {% endif %}
+            {% elif application is not none and review is not none %}
+                <br />
+                {% if review.status == "Accepted" %}
```

Comment:
I think we should add an extra section explicitly for when they have been accepted, but the RSVP deadline has passed. The page currently looks like this when that's the case:

![image](https://user-images.githubusercontent.com/26036279/95659838-dad94400-0af1-11eb-9ccb-be07ab309df4.png)

It would be best if the content could change to reflect that the deadline has passed, or at least change the application status to "RSVP deadline has passed".